test/ajv.spec.ts

```
@@ -0,0 +1,339 @@
+import { validatePayload } from '../src/ajv';
+import { ovmRewardsSplitPayloadSchema, ovmTotalSplitPayloadSchema } from '../src/schema';
+import { OVMRewardsSplitPayload, OVMTotalSplitPayload } from '../src/types';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)